<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParrotShare</class>
 <widget class="QMainWindow" name="ParrotShare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParrotShare</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>330</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabConfiguration">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QLabel" name="confLabelDownloadFolder">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Download folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="confLabelCacheFolder">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cache folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="confLabelTcpPort">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen to port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="confFieldDownloadFolder">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>330</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="confFieldCacheFolder">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>330</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="confFieldTcpPort">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>60</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="confButtonSetDownloadFolder">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confButtonSetCacheFolder">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFriends">
     <attribute name="title">
      <string>Friends</string>
     </attribute>
     <widget class="QListView" name="friendsList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>180</width>
        <height>240</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="friendsButtonAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="friendsButtonDelete">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="friendsLabelSharedSecret">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Shared Secret</string>
      </property>
     </widget>
     <widget class="QLabel" name="friendsLabelAddress">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="friendsLabelPort">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="friendsLabelStatus">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="friendsFieldSharedSecret">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>230</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="friendsFieldAddress">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>230</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="friendsFieldTcpPort">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>60</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="friendsFieldStatus1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>230</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="friendsFieldStatus2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>135</y>
        <width>230</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="friendsLabelShares">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Shares</string>
      </property>
     </widget>
     <widget class="QTableView" name="friendsTableShares">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>230</width>
        <height>125</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="friendsButtonShareRW">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>R/W</string>
      </property>
     </widget>
     <widget class="QPushButton" name="friendsButtonShareR">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="friendsButtonShareOff">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="friendsCheckboxConnect">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="friendsButtonPing">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ping</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabShares">
     <attribute name="title">
      <string>Shares</string>
     </attribute>
     <widget class="QListView" name="sharesList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>180</width>
        <height>240</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sharesButtonAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sharesButtonDelete">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTableView" name="sharesTableFriends">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>230</width>
        <height>125</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="sharesButtonShareOff">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sharesButtonShareR">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sharesButtonShareRW">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>R/W</string>
      </property>
     </widget>
     <widget class="QLabel" name="sharesLabelFriends">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Friends</string>
      </property>
     </widget>
     <widget class="QLabel" name="sharesLabelDir">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sharesFieldPath">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>20</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sharesButtonSetPath">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabUnused">
     <attribute name="title">
      <string>Unused</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_and_Quit"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_and_Quit">
   <property name="text">
    <string>Save and Quit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ParrotShare</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
